index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="styles/home.css" />
+    <title>Friki Corner</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="desktop flexrowDesk navsup">
+          <img src="Img/FaceIcon.png" alt="Facebook" />
+          <img src="Img/TwitterIcon.png" alt="Twitter" />
+          <ul class="ul flexrow">
+            <li>Logout</li>
+            <li>MyPage</li>
+            <li>Mexico</li>
+          </ul>
+        </div>
+        <div class="desktop flexrowDesk navdesk">
+          <div class="divop flexrowDesk">
+            <a href="index.html"><img src="Img/logo.png" alt="Friki Corner" /></a>
+            <ul class="ul flexrow">
+              <li><a class="select" href="pages/about.html">About</a></li>
+              <li class="select">Program</li>
+              <li class="select">Join</li>
+              <li class="select">Sponsor</li>
+              <li class="select">News</li>
+              <li><button>Friki Corner</button></li>
+            </ul>
+          </div>
+        </div>
+        <div class="mobmenu flexrow">
+          <div class="mobile">
+            <img id="hambur" src="Img/mobile_menu.png" alt="Menu icon" />
+          </div>
+        </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As the below detail design of your `header` was not pointed out by the previous reviewer. I think it would be better if you fix it by using the correct spacing like on the template. This make your application stick as much as possible to the original design.
### Template Design
![Screenshot from 2023-05-24 01-55-21](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/83042742/f3ee4ac0-0587-4ca2-9d58-31dcf2c40bac)

### Your Design
![Screenshot from 2023-05-24 01-54-33](https://github.com/AndreaM2429/Capstone_project_Conference_page/assets/83042742/92bcd802-e5a9-4f74-8299-0aee446afc1a)